### ISSUE TITLE
Question: 06_Neural_../05_implement_.. , about the function tf.nn.conv2d()

### ISSUE THREAD

**Issue by leigaoyi** [nfmcclure/tensorflow_cookbook#39]
**tf.nn.conv2d(input,filter,strides,padding)** is the function of convolutional network in tensorflow.
Here, the author says the input of 4-D data should have the size **[batch#, width, height, channels]**,
but the official web gives the description:
 **input tensor of shape [batch, in_height, in_width, in_channels]**
Will this make difference , if we replace the place of width and height in our experience?

**Comment by nfmcclure** [comment]
This should not make a difference.  If you need to keep track of `height` and `width` in the scripts, you can also transpose the images/tensors in those dimensions as well.  Let me know if you run into any problems with this.
